corehq/apps/auditcare/models.py

```diff
@@ -212,7 +211,7 @@ def audit_login(cls, request, user, *args, **kwargs):
         audit = cls.create_audit(cls, user)
         audit.ip_address = get_ip(request) or ''
         audit.http_accept = request.META.get('HTTP_ACCEPT')
-        audit.path_info = request.META.get('PATH_INFO', '')
+        audit.path = request.META.get('PATH_INFO', '')
```

Comment:
I refactored `audit.path_info` to `audit.path` and moved it to the base class (probably in a later commit than the one you're commenting on). I did not change the `request.META` field we're using since that seemed like an out of scope refactor. Could do it in a follow-up PR if you think it's important.

Out of curiosity, what is the difference between `request.META.get('PATH_INFO', '')` and `request.META.get('PATH', '')`? Is `PATH_INFO` deprecated or something?

Reply:
From the [docs](https://docs.djangoproject.com/en/3.1/ref/request-response/#django.http.HttpRequest.path_info): `For example, if the WSGIScriptAlias for your application is set to "/minfo", then path might be "/minfo/music/bands/the_beatles/" and path_info would be "/music/bands/the_beatles/"`. I'm guessing the distinction probably doesn't matter for our application? But neither are deprecated, and they do mean different things.

Reply:
Interesting. It shouldn't matter for our application since I think `PATH` and `PATH_INFO` will always be the same (we do not set `WSGIScriptAlias`).